index.html

```diff
@@ -318,41 +318,31 @@ <h3 id="unacceptablebehavior">
           <li>Use of coded language (also known as "dog whistles") used to
           rally support for hate groups or to intimidate vulnerable groups.
           </li>
+          <li>Patronizing language or behavior:
+            <ul>
+              <li>Assuming that people or groups need concepts defined or explained to them. 
+              </li>
+              <li>Assuming that particular groups of people are technically unskilled (e.g., “So easy your grandmother could do it”).
+              </li>
+              <li>Assuming your experience or knowledge on a topic is essential to a discussion, even if it takes discussion off topic or interrupts the flow of the conversation.
+              </li>
+            </ul>
+          </li>
           <li>Microaggressions, which are small comments or questions, either
           intentional or unintentional, that marginalize people by
           communicating hostile, derogatory, or negative beliefs. Examples
           include:
             <ul>
-              <li>Patronizing language or behavior:
-                <ul>
-                  <li>Be aware that, regardless of the speaker's intentions,
```

Comment:
I'm concerned about dropping this bullet entirely.  I think point out that there are some comment triggers/identifiers of patronizing language can be illuminating.  I probably used to say "well actually" a lot.  I don't anymore.  :)

Reply:
I also think the "Well, actually" example is useful to include, because it helps to alert the reader to the existence of terms that may be neutral to the speaker, but nonetheless are triggers to some groups.  

Since it is not possible to know all trigger phrases for all possible groups, I think it is helpful to sensitize the reader to be on the lookout for them.  That example helped to raise my awareness in a way that a general statement would not.

Reply:
I removed "well, actually" because:
1. It's not the only trigger phrase, but listing them all would be impossible, and not particularly helpful
2. I can name more times I've been personally patronized to without the use of that phrase than with it
3. In my experience with CEPC, putting in examples of things tends to lead to people using those examples as excuses for their behaviour ("I can't have been patronizing, it's not like I said 'well, actually...'")

Reply:
> ["Well, actually" is] not the only trigger phrase, but listing them all would be impossible, and not particularly helpful

I agree that listing all of them would be impossible, and I can see that they don't help with the enforcement role of the document, but I think one or two examples are still helpful for educational purposes -- _especially_ examples (like "Well, actually") that may not be widely known, though I think it's important to explain them.  I think that helps alert the reader to be on the lookout for new ones that the reader may not know about.  Both the "Well, actually" and "thug" examples were eye opening to me (as one data point).  And once I researched those terms on the web, it was clear that I was not at all alone in being unaware of their offensive interpretation to some.

> I can name more times I've been personally patronized to without the use of that phrase than with it.

Agreed.

> In my experience with CEPC, putting in examples of things tends to lead to people using those examples as excuses for their behaviour ("I can't have been patronizing, it's not like I said 'well, actually...'")

Interesting point.  But rather than removing examples, how about addressing that issue head-on?  How about adding something like the following in the list of Unacceptable behaviors:

> Feigning a lack of understanding of this code, such as pretending not to understand that examples of unacceptable behavior are merely examples, and do not constitute an exhaustive list of unacceptable behaviors.